parlai/agents/repeat_query/repeat_query.py

```diff
@@ -31,3 +31,12 @@ def act(self):
             reply['text'] = 'Nothing to repeat yet.'
         reply['episode_done'] = False
         return Message(reply)
+
+    def batch_act(self, observations):
+        batch_reply = []
+        original_obs = self.observation
```

Comment:
Why should we save the `original_obs`? Not quite sure if I understand the logic here.

Reply:
I just don't want batch_act to change the `observation` attribute of the instance. So i just keep it as whatever it was before using a temporary variable `original_obs`. I didn't know what else to do.

Reply:
If folks are using `batch_act`, I don't think they'll likely be calling `observe`/`act` on the agent via some other codepath (since I'd assume they'd just use one or the other in say, a batch world and not be mixing + matching the two).

Not a big deal, but as a comment.